### PR TITLE
Add dependency references for unit tests

### PR DESCRIPTION
**Pull request by abdullahselek**
- Dependency references added in Podfile with using CocoaPods
- Workspace created and travis config file updated with new changes

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,4 +25,8 @@ xcuserdata/
 *.hmap
 *.ipa
 *.dSYM.zip
-*.dSYM
\ No newline at end of file
+*.dSYM
+
+# CocoaPods
+Pods/
+Podfile.lock
```
Comment (sschuberth) [inline .gitignore]: How about using the opportunity to add a final newline?

File: Podfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
+platform :ios, '9.0'
```
Comment (sschuberth) [inline Podfile:1]: Shouldn't that be '9.3' like in `.travis.yml`?

File: Podfile [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+target 'OnDemandPassenger' do
+  use_frameworks!
+
+  pod 'Quick', '1.1.0'
+  pod 'OCMock', '3.4'
+  pod 'Expecta', '1.0.5'
+  pod 'OHHTTPStubs', '6.0.0'
+
```
Comment (sschuberth) [inline Podfile]: Superfluous blank line.

File: Podfile [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+end
+
+post_install do |installer|
+  installer.pods_project.targets.each do |target|
+    target.build_configurations.each do |config|
+      config.build_settings['SWIFT_VERSION'] = '3.0'
+    end
+  end
+end
```
Comment (sschuberth) [inline Podfile]: How about using the opportunity to add a final newline?

**Review by sschuberth — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (abdullahselek) [reply .gitignore]: Done

File: Podfile (hunk 2 above)
Comment (abdullahselek) [reply Podfile:1]: It is just for Travis because of missing simulator version

File: Podfile (hunk 4 above)
Comment (abdullahselek) [reply Podfile]: Done

**Review by sschuberth — APPROVED** [review]

**Pull request merged**